tests/network/vmi_multus.go

```diff
@@ -667,7 +667,6 @@ var _ = Describe("[Serial]SRIOV", func() {
 	tests.BeforeAll(func() {
 		virtClient, err = kubecli.GetKubevirtClient()
 		util.PanicOnError(err)
-		tests.SkipIfNonRoot(virtClient, "SRIOV")
```

Comment:
doesnt it mean now that it will run it on both lanes ?
(the regular one, and the non-root one)
i think it means that and it shouldn't run it on the regular one, its also what i understand that meaning of this PR according the PR description

Reply:
Note that this skip shouldn't be there in the first place as there is dedicated lanes for non-root setup and tests suite

Reply:
> Note that this skip shouldn't be there in the first place as there is dedicated lanes for non-root setup and tests suite

but i see the skip on the regular lanes as well (or at least saw last week unless something changed)

Reply:
I think the description of the PR and actual PR don't match. 

The description is from `tests/nonroot_test.go:19`

Reply:
Changing this PR to draft until we decide what to do

Reply:
>Actually it will run it and skip it every time due to NonRoot feature gate is missing,
and work only on non-root lanes where we enable this feature.

Can you elaborate? I don't think this can happen